lib/node-package-manager.ts

```diff
@@ -48,30 +24,54 @@ export class NodePackageManager implements INodePackageManager {
 			}
 
 			try {
-				return this.loadAndExecute("install", [pathToSave, packageName], { config: config }).wait();
+				let jsonContentBefore = this.$fs.readJson(path.join(pathToSave, "package.json")).wait();
+				let dependenciesBefore: Array<string> = [];
+				if(jsonContentBefore && jsonContentBefore.dependencies) {
+					for(let name in jsonContentBefore.dependencies) {
+						dependenciesBefore.push(name);
+					}
+				}
+
+				let flags = this.getFlagsString(config, true);
+				let params = ["install"];
+				if(packageName !== pathToSave) {
+					params.push(packageName); //because npm install ${pwd} on mac tries to install itself as a dependency (windows and linux have no such issues)
+				}
+				params = params.concat(flags);
+				this.$childProcess.spawnFromEvent("npm", params, "close", { cwd: pathToSave }).wait();
```

Comment:
How does this code work on Windows? On Mac and Linux the executable name is `npm`, but on Windows it's `npm.cmd`